src/ray/raylet/main.cc

```diff
@@ -215,6 +215,12 @@ int main(int argc, char *argv[]) {
                        << object_manager_config.rpc_service_threads_number
                        << ", object_chunk_size = "
                        << object_manager_config.object_chunk_size;
+        // Initialize stats.
```

Comment:
Any particular reason why you move it up here?

Reply:
@rkooo567 we'd better make sure rayconfig is initialized and it's done in async callback. If we put this init operation outside, we may access a config value before Rayconfig updating completely.